src/test/java/com/commercetools/sync/categories/helpers/CategorySyncStatisticsTest.java

```diff
@@ -95,23 +97,25 @@ public void getStatisticsAsJSONString_WithoutCalculatingProcessingTime_ShouldGet
         categorySyncStatistics.incrementUpdated();
         categorySyncStatistics.incrementProcessed();
 
-        final String statisticsAsJsonString = getStatisticsAsJSONString(categorySyncStatistics);
+        final String statisticsAsJsonString = getStatisticsAsJsonString(categorySyncStatistics);
         assertThat(statisticsAsJsonString)
-                .isEqualTo("{\"reportMessage\":\"Summary: 3 categories were processed in total (1 created, 1 updated and 1 categories failed to sync).\",\"" +
-                        "updated\":1,\"" +
-                        "created\":1,\"" +
-                        "failed\":1,\"" +
-                        "processed\":3,\"" +
-                        "processingTimeInDays\":" + categorySyncStatistics.getProcessingTimeInDays() + ",\"" +
-                        "processingTimeInHours\":" + categorySyncStatistics.getProcessingTimeInHours() + ",\"" +
-                        "processingTimeInMinutes\":" + categorySyncStatistics.getProcessingTimeInMinutes() + ",\"" +
-                        "processingTimeInSeconds\":" + categorySyncStatistics.getProcessingTimeInSeconds() + ",\"" +
-                        "processingTimeInMillis\":" + categorySyncStatistics.getProcessingTimeInMillis() + ",\"" +
-                        "humanReadableProcessingTime\":\"" + categorySyncStatistics.getHumanReadableProcessingTime() + "\"}");
+                .isEqualTo("{\"reportMessage\":\"Summary: 3 categories were processed in total (1 created, 1 updated "
```

Comment:
yes, report is just for human readability purposes. The other one in the JSON KV pairs might be used for plotting graphs of the stats on the logging tool (logentries for example).